app/actions/CustomActionBuilders.scala

```diff
@@ -40,20 +40,14 @@ class CustomActionBuilders(
   // Prevents the identity validation email sending users back to our checkout.
   private val idSkipValidationReturn: (String, String) = "skipValidationReturn" -> "true"
 
-  private def idWebAppRegisterUrl(path: String, clientId: String, idWebAppRegisterPath: String, campaignCode: Option[String]): String =
+  private def idWebAppRegisterUrl(path: String, clientId: String, idWebAppRegisterPath: String): String =
     idWebAppUrl / idWebAppRegisterPath ?
       ("returnUrl" -> s"$supportUrl$path") &
       idSkipConfirmation &
       idSkipValidationReturn &
-      "clientId" -> clientId &
-      "INTCMP" -> campaignCode
-
-  def chooseRegister(identityClientId: String): RequestHeader => Result = request => {
-    SeeOther(idWebAppRegisterUrl(request.uri, identityClientId, "register", Some("recurring-contributions-control")))
-  }
-
+      "clientId" -> clientId
   def chooseSignInStart(identityClientId: String): RequestHeader => Result = request => {
```

Comment:
Does it make sense to drop the word `choose` from the front of this method name?

Reply:
before we had this test, it was called chooseRegister. I did think about this and even that didn't make sense. I'm going to rename it to onUnauthenticated, as that describes it's purpose better